*** tests/A-09-06-M-ok.out	2021-10-14 18:14:02.630747800 -0300
--- tests/A-09-06-M-ok.outhyp	2021-11-05 14:36:51.023224000 -0300
***************
*** 49,63 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: aaaaaaaaaaaaa|aa|aa|NORMAL|0|0|0|0
! AAAAS1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  BBBBW2|Pedraria Fonseca|Oeiras, Portugal|NORMAL|0|0|0|0
  CON|Lages do Chão-con|Lisboa, Portugal|NORMAL|0|0|0|0
  DDDDDDP3|ObBrian Rocks|Koln, Germany|NORMAL|0|0|0|0
- dede|de de|de@de|NORMAL|0|0|0|0
- eeEE|ee EE|e@e|NORMAL|0|0|0|0
  EEEEEER2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
  EFFFFF4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
  yyy|yyy|yyyyyy|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 49,63 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: AAAAS1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  BBBBW2|Pedraria Fonseca|Oeiras, Portugal|NORMAL|0|0|0|0
  CON|Lages do Chão-con|Lisboa, Portugal|NORMAL|0|0|0|0
  DDDDDDP3|ObBrian Rocks|Koln, Germany|NORMAL|0|0|0|0
  EEEEEER2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
  EFFFFF4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
+ aaaaaaaaaaaaa|aa|aa|NORMAL|0|0|0|0
+ dede|de de|de@de|NORMAL|0|0|0|0
+ eeEE|ee EE|e@e|NORMAL|0|0|0|0
  yyy|yyy|yyyyyy|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
