*** tests/A-11-01-M-ok.out	2021-11-07 13:04:59.127560300 -0300
--- tests/A-11-01-M-ok.outhyp	2021-11-11 22:31:22.307775500 -0300
***************
*** 28,34 ****
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|1|0
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 28,34 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|1|10
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 96,102 ****
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|100|10
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 96,102 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|2|5000
  ROLHA|2|500
! SAL|100|20
  VIDRO|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 105,110 ****
--- 105,111 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do produto: SAL|M1|100|10
+ SAL|S1|1|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
