*** tests/A-12-01-M-ok.out	2021-11-07 13:04:59.218638100 -0300
--- tests/A-12-01-M-ok.outhyp	2021-11-11 18:28:43.043578000 -0300
***************
*** 9,16 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 9,15 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
