*** tests/A-16-02-M-ok.out	2021-11-07 13:04:59.283344400 -0300
--- tests/A-16-02-M-ok.outhyp	2021-11-12 13:13:27.397333200 -0300
***************
*** 94,100 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|250|0|25|25
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 94,100 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
